go/pkg/apis/paddlepaddle/v1alpha1/types.go

```diff
@@ -92,20 +101,28 @@ type TrainerJobScaleStatus struct {
 type TrainingResourceType string
 
 const (
-	MASTER  TrainingResourceType = "MASTER"
+	// MASTER is the master name of TrainingResourceType.
+	MASTER TrainingResourceType = "MASTER"
+	// PSERVER is the pserver name of TrainingResourceType.
 	PSERVER TrainingResourceType = "PSERVER"
+	// TRAINER is the trainer name of TrainingResourceType.
 	TRAINER TrainingResourceType = "TRAINER"
 )
 
 // ResourceState is the state of a type of resource
 type ResourceState string
 
 const (
-	ResourceStateNone      ResourceState = ""
-	ResourceStateStarting                = "starting"
-	ResourceStateRunning                 = "running"
-	ResourceStateFailed                  = "failed"
-	ResourceStateSucceeded               = "succeeded"
+	// ResourceStateNone is the none name of ResourceState.
+	ResourceStateNone ResourceState = ""
+	// ResourceStateStarting is the starting name of ResourceState.
+	ResourceStateStarting = "starting"
+	// ResourceStateRunning is the  running name of ResourceState.
+	ResourceStateRunning = "running"
+	// ResourceStateFailed is the failed name of ResourceState.
+	ResourceStateFailed = "failed"
+	// ResourceStateSucceeded is the succeeded name of ResourceState
+	ResourceStateSucceeded = "succeeded"
```

Comment:
ResourceStateSucceeded is the succeeded state of training job

Reply:
Done.